Docs 9548 editorial review messages esb (#105)
mulesoft/docs-partner-manager: modules/ROOT/pages/delete-identifiers.adoc

= Delete Identifiers

> When an identifier is no longer needed, consider deleting the identifier.

Reviewer: Why? I mean, I'm tidy, so I would naturally do this, lol, but I think we should give the audience a reason to do everything we ask them to do. GLOBAL

Author: Done

You can delete identifiers such as X12 ISA, X12 GS, AS2, and DUNS.

> Before you delete an identifier, consider the following:

Reviewer: Consider moving any or all of these into the intro text itself. Personally, I kind of like the idea of an intro something like "Although you must always have at least one identifier per message flow, you can delete identifiers that are no longer used in that message flow [and that you no longer need? Is there a difference?] so that <reason for deleting>"

Then I would go to the steps...because the title here implies that this text is the task, rather than "Deleting."

Then I would mention the message routing thing...or maybe even include that before the steps. 

I'm not sure you need the final bullet, but maybe end with that warning and an explanation?

Another option is to consider prerequisites before Step 1.

Reviewer: oops...I just can't control my CU impulses! Back to CE....

Author: Adjusted slightly - left list, added Steps heading, and reworded intro. PM wants the info in the way it was presented.

* Remember that there must always be at least one identifier per message flow.
* You cannot delete an identifier that is used in a message flow.
* Deleting an existing host or partner identifier impacts the identification and routing of B2B messages to the associated partner.
* If you need to replace the identifier with another identifier, record the required values and qualifier type.

> == Deleting Identifiers

Reviewer: ```suggestion
```

Task (steps) titles must use imperative voice.

. In the left navigation menu, select *Partners*.

> . From the list of partners, select the partner containing the identifier you want to delete.

Reviewer: I don't think we need to specify the list; I would assume that selecting *Partners* takes me to something like a list, and the user is in the UI after all:
```suggestion
. Select the partner containing the identifier you want to delete.
```

> . In the *Identifiers* section, in the right-hand side of the *Identifiers* row, hover over the identifier you want to delete and click the trash can.

Reviewer: Is the UI so clumsy that we have to designate "right side"? Doing so is an accessibility no-no, but if the UI is confusing....
Otherwise:
```suggestion
. In the *Identifiers* row, hover over the identifier you want to delete and click the trash can icon.
```
It would be swell if we could capture the icon and present it instead of naming it.

> . If you want to replace the identifier, click *New* and xref:partner-manager-identifiers.adoc[follow the steps to create an identifier].

Reviewer: Decide whether you want to make this task Delete or Delete or Replace.
In the current context, I think replace gets lost and is off topic according to the title (see Line 12 comment).

Another option is to retitle the Create topic Create or Replace?

Author: Done - leaving the explanation in the prereq and moving the link to the See Also section.

> After successfully deleting an identifier, you can create a new identifier with the same name as the one you deleted.

Reviewer: Is the point here that you can reuse the name?
Otherwise...why?

Author: Deleted. This is pretty standard and doesn't need to be said.
